### PR TITLE
GEODE-5499: Changing testTxRmtMirror to use awaitility, not 200 ms sleep

### PR DESCRIPTION
**Pull request by upthewaterspout**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by upthewaterspout** [comment]
@mhansonp @balesh2

File: geode-core/src/integrationTest/java/org/apache/geode/cache30/MultiVMRegionTestCase.java [hunk 1]
```diff
@@ -7306,6 +7308,34 @@ public void run() {
             }
           }
         };
+
+
+
+    Region rgn;
```
Comment (WireBaron) [inline geode-core/src/integrationTest/java/org/apache/geode/cache30/MultiVMRegionTestCase.java]: Can you just call this 'region'?

File: geode-core/src/integrationTest/java/org/apache/geode/cache30/MultiVMRegionTestCase.java [hunk 2]
```diff
@@ -7306,6 +7308,34 @@ public void run() {
...
+    // and dropped in a remote non-mirror
+    VM vm0 = VM.getVM(0);
+    VM vm1 = VM.getVM(1);
+    vm0.invoke(createMirror);
+    vm1.invoke(createNonMirror);
+    rgn = createRegion(rgnName);
+
+    txMgr.begin();
+    rgn.create("key", "value");
+    // TransactionId txId = txMgr.getTransactionId();
```
Comment (WireBaron) [inline geode-core/src/integrationTest/java/org/apache/geode/cache30/MultiVMRegionTestCase.java]: Dead code

**Comment by upthewaterspout** [comment]
@galen-pivotal - This test is in a base class that is extended by a whole bunch of other tests that configure different region types - so this is run with either ack or no-ack.

The test has seen failures for some time, but it may be worse now with the newer infrastructure.

**Pull request merged**